DAO.sol

```diff
@@ -821,7 +822,10 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
 
     function halveMinQuorum() returns (bool _success) {
-        if (lastTimeMinQuorumMet < (now - quorumHalvingPeriod)) {
+        // this can only be called after `quorumHalvingPeriod` has passed or at anytime
+        // by the curator with a delay of at least `minProposalDebatePeriod` between the calls
+        if ((lastTimeMinQuorumMet < (now - quorumHalvingPeriod) || msg.sender == curator)
+			&& lastTimeMinQuorumMet < (now - minProposalDebatePeriod)) {
```

Comment:
Well then I could do the same using slockits address and have you guys removed from the list. 

The attack still works if the first part is excluded and there is just 1 proposals. That does not reach the quorum.

Basically i don't like that the quorum can be changed so quickly. which changes a proposals that didn't reach the quorum to one that passed.